### ISSUE TITLE
Issue when using org-ml-build-table! with a list of lists (for multiple rows)

### ISSUE THREAD

**Issue by arozbiz** [ndwarshuis/om.el#43]
The documentation for org-ml-build-table! says that "row-lists is a list of lists," but when I try to use a list of lists I get an error. E.g., `(org-ml-build-table! '(("R" "A") ("G" "E")))` throws an error, whereas `(apply #'org-ml-build-table! '(("R" "A") ("G" "E")))` works as expected (as does the example in the documentation).

**Comment by ndwarshuis** [comment]
How about this?

```
(org-ml-build-table! &key TBLFM POST-BLANK &rest ROW-LISTS)

Return a new table node.

Each member of ROW-LISTS will be converted
to a table-row node via ‘org-ml-build-table-row!’ (see that function for
restrictions).
```

**Comment by arozbiz** [comment]
Yes, that would fix the documentation. Though I wonder if a better solution might be to modify the function so as to accept a list of lists as a single ROW-LISTS parameter. That would avoid having to use the `apply` formulation I describe above.

**Comment by ndwarshuis** [comment]
I could do that but all `*build` function in the library are written with the convention that child members are variable-number arguments to be consumed by a `&rest` parameter. I don't have any philosophical objection to the alternative (a single list as you mentioned) since either will ultimately make some things harder and others easier, but nonetheless the convention I went with in the beginning was the former and I would rather not mix them.

In your example, what's wrong with `(org-ml-build-table! '("R" "A") '("G" "E"))`?

**Comment by arozbiz** [comment]
Totally makes sense to have a single convention. The only issue with `(org-ml-build-table! '("R" "A") '("G" "E"))` is that it makes it hard to generate the rows at runtime (without using the `apply` formulation).

**Comment by ndwarshuis** [comment]
fixed with 75a5569
